src/scss/skin-modern/components/_quickseekbutton.scss

```diff
@@ -0,0 +1,18 @@
+@import '../variables';
+@import '../mixins';
+
+.#{$prefix}-ui-quickseekbutton {
+  @extend %ui-button;
+
+  &:hover {
+    @include svg-icon-shadow;
+  }
+
+  &[data-#{$prefix}-seek-direction='forward'] {
+    background-image: url('../../assets/skin-modern/images/quickseek-fastforward.svg');
+  }
+
+  &[data-#{$prefix}-seek-direction='rewind'] {
+    background-image: url('../../assets/skin-modern/images/quickseek-rewind.svg');
+  }
```

Comment:
Do we need any specific styling for the mobile UI variant?

Reply:
IMO there is nothing special needed for now.